src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
Perhaps we can come up with a better method name than `length`. This is incredibly confusing, IMHO... Maybe `numElements` or `count`.